src/SUMMARY.md

```diff
@@ -26,6 +26,7 @@
     * [Slash Commands](guide/advanced-guides/slash-commands.md)
   * [Extras](guide/extras/README.md)
     * [Custom Events](guide/extras/custom-events.md)
+    * [Voice](guide/extras/music.md)
```

Comment:
This should be @akarui/aoi.music, since this entire guide is related towards it.

Reply:
it's  more related to Voice class cause if i add aoi.music then i have to explain all classes